pkg/resource/slo/delete_test.go

```diff
@@ -29,10 +29,25 @@ import (
 
 	libAPI "github.com/dynatrace/dynatrace-configuration-as-code-core/api"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/idutils"
-	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/delete/internal/slo"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/delete/pointer"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/resource/slo"
 )
 
+type deleteStubClient struct {
+	called bool
```

Comment:
nit: I know it was also called lke this before but `called` may be a bit misleading. `deleteCalled` would fit better here.